index.rst

```diff
@@ -15,5 +10,234 @@ Change Record
+
+Baseline Documents
+------------------
+
+The baseline design and architecture of the DM Subsystem is captured through a set of change-controlled
+documents, including:
+
+* LDM-148 DM System Design
+
+* LDM-151 DM Applications Design
+
+* LDM-152 DM Middleware Design
+
+* LDM-135 DM Database Design
+
+* LDM-131 DM SUIT Conceptual Design
+
+The complete set of DM baseline documents can be found in Docushare, Collection-2511.
+
+All together, this collection of documents capture all work DM has to deliver during the LSST Construction.
+
+Planning Packages
+-----------------
+The work is divided into a few hundred concrete, technical *packages of planned work*. The packages divide work
+at the 4th Level of WBS, e.g., typically a 4th Level WBS is broken down into 20-50 packages. Each such package
+addresses a piece of work defined in the baseline documents, and all together they cover the entire DM baseline.
+Each package has a clearly defined scope, deliverable(s), resource cost, and an end date. Additionally,
+each package keeps track of which section(s) of which baseline document it implements.
+
+
+
+In the EVMS world, these packages are implemented directly as *Planning Packages* (PP).
+
+Milestones
+----------
+All significant development stages of each planning package are tracked through *milestones*.
+Each planning package has at least one milestone, used to mark the completion of a given package.
+It is not uncommon to have multiple milestones per package to track progress along the way.
+Milestones can be related to multiple planning packages, potentially from different WBSes.
+Each milestone has a description, and due date; it has no duration, and no resources assigned to it.
+
+Each activity within a planning package that involves a cross-team dependency is required to end with
+a milestone; this includes activities related to delivering software components, hardware, and services.
+
+Further, milestones have *levels*:
+
+* Level 1 denotes the most important milestones exposed at the NSF level
+
+* Level 2 denotes cross-subsystem milestones (for example, DM milestones that affect the Camera Subsystem)
+
+* Level 3 denotes cross-team milestones within a subsystem (for example, Middleware milestones that affect the DRP Team)
+
+* Level 4 denotes internal milestones within a team.
+
+Relationships between milestones, as well as between milestones and planning packages are captured:
+milestones typically *block* planning packages, milestones can *block* other milestones, or they
+can *relate* to other milestones.
+
+Baseline Long-Term Plan
+-----------------------
+
+Planning packages, together with milestones form the *Baseline Plan*.
+This plan is under change control.
+As such, most changes to the plan, including changing scope, cost or schedule, must be approved by the appropriate body.
+Specifically:
+
+* Change Control Board approval is required for any non-trivial change to a planning package or Level 1 or 2 milestone;
```

Comment:
Technically, the Change Control Board does not "approve" anything.  It makes recommendations to the LSST project manager.  

I am concerned that change requests that look like WBS and budget items have in recent times often been treated as "CCB information items" that are not even subjected to the full CCB process - no discussion unless someone raises a flag, and no recorded vote.

Particularly in the case of computing-related issues (whether software or systems) the only way any real scrutiny from the CCB can be obtained is by flagging an item as requiring non-routine attention.